tests/test_utils.py

```diff
@@ -0,0 +1,27 @@
+from unittest.mock import *
```

Comment:
Importing with `* ` is considered a bad practice and usually discouraged in lintiing tools, only import what is needed, which is `patch`

Reply:
Fixed here d3bd146